relationshiptypes.md

```diff
@@ -68,3 +68,71 @@ A sample can be found at https://dspace7-entities.atmire.com/rest/#https://dspac
 ```
 
 The 2 [item types](itemtypes.md) are embedded
+
+## Search methods
+
+### Relationship types containing an entity type
+**/api/core/relationshiptypes/search/byEntityTypeId?id=<:entity-type-id>**
+
+Parameters:
+* The `id` should be the entity type id from the [entity types endpoint](entitytypes.md). It is mandatory. It can occur on either the left or right hand side
+
+A sample search would be /server/api/core/relationshiptypes/search/byEntityTypeId?id=1
+
+It would respond with
+```json
+{
+  "_embedded": {
+    "relationshiptypes": [
+      {
+        "id": 10,
+        "leftwardType": "isAuthorOfPublication",
+        "rightwardType": "isPublicationOfAuthor",
+        "copyToLeft": false,
+        "copyToRight": false,
+        "leftMinCardinality": 0,
+        "leftMaxCardinality": null,
+        "rightMinCardinality": 0,
+        "rightMaxCardinality": null,
+        "type": "relationshiptype"
+      },
+      {
+        "id": 1,
+        "leftwardType": "isAuthorOfPublication",
+        "rightwardType": "isPublicationOfAuthor",
+        "copyToLeft": false,
+        "copyToRight": false,
+        "leftMinCardinality": 0,
+        "leftMaxCardinality": null,
+        "rightMinCardinality": 0,
+        "rightMaxCardinality": null,
+        "type": "relationshiptype"
+      }
+    ]
+  }
+}
+```
+
+## Property-based projections
+
+[Property-based projections](projections.md#property-based-projections) can add new JSON properties to the response. When requesting the projection, any `relationshiptype` in the response will add these properties.  
+The REST request doesn't have to be on the `relationshiptype` endpoint directly, it can be on another endpoint which simply embeds relationship types.
+
+### Verify whether a given entity type is the left or right
+**?projection=CheckSideEntityInRelationshipType&checkSideEntityInRelationshipType=<:entitytype>**
```

Comment:
Just a quick, and minor, question: why in this projection entitytype name is used as possible value for checkSiedEntityInRelationshipType, while on above call /api/core/relationshiptypes/search/byEntityTypeId the entity type id is used? 
If possible, I'd use id or entity type name in both calls.

Reply:
These are two equivalent requests, depending on whether you currently have the name of the id of the given entity type.
I don't know for sure whether we need both in Angular, but they can both be useful. Because there's little extra effort in implementing both, I just added them